ui/raidboss/data/05-shb/eureka/zadnor.js

```diff
@@ -81,30 +133,1155 @@ export default {
+        },
+      },
+    },
+    {
+      id: 'Zadnor Saunion Anti-Personnel Missile',
+      netRegex: NetRegexes.startsUsing({ source: 'Saunion', id: '5DC2' }),
+      condition: Conditions.targetIsYou(),
+      response: Responses.spread(),
+    },
+    {
+      id: 'Zadnor Saunion Missile Salvo',
+      netRegex: NetRegexes.startsUsing({ source: 'Saunion', id: '5DC3' }),
+      response: Responses.stackMarkerOn(),
+    },
+    {
+      id: 'Zadnor Saunion Wildfire Winds',
+      netRegex: NetRegexes.startsUsing({ source: 'Dawon The Younger', id: '5DCD', capture: false }),
+      delaySeconds: 10,
+      infoText: (_data, _matches, output) => output.text(),
+      outputStrings: {
+        text: {
+          // TODO: during spiral scourge could be "get under middle/outer light orb"?
+          en: 'Get Under Light Orb',
+          de: 'Unter einem Lichtorb stellen',
+          fr: 'Allez sous un Orbe lumineux',
+          ja: '白玉へ',
+          cn: '靠近白球',
+          ko: '하얀 구슬 안으로',
+        },
+      },
+    },
+    {
+      id: 'Zadnor Saunion Tooth and Talon',
+      netRegex: NetRegexes.startsUsing({ source: 'Dawon The Younger', id: '5DD4' }),
+      condition: tankBusterOnParty(),
+      response: Responses.tankBuster(),
+    },
+    {
+      id: 'Zadnor Saunion Swooping Frenzy',
+      netRegex: NetRegexes.startsUsing({ source: 'Dawon The Younger', id: '5DD0', capture: false }),
+      infoText: (data, _matches, output) => {
+        // Every other Swooping Frenzy is followed by a Frigid Pulse, starting with the first.
+        data.saunionSwoopingCount = (data.saunionSwoopingCount || 0) + 1;
+        if (data.saunionSwoopingCount % 2)
+          return output.text();
+      },
+      outputStrings: {
+        text: {
+          en: 'Follow Dawon',
+        },
+      },
+    },
+    {
+      id: 'Zadnor Diablo Advanced Death Ray',
+      netRegex: NetRegexes.headMarker({ id: '00E6' }),
+      condition: (data) => data.ce === 'dalriadaDiablo',
+      // TODO: this is maybe worth promoting to responses?
+      response: (data, matches, output) => {
+        // cactbot-builtin-response
+        output.responseOutputStrings = {
+          tankLaserOnYou: {
+            en: 'Tank Laser on YOU',
+          },
+          avoidTankLaser: {
+            en: 'Avoid Tank Laser',
+          },
+        };
+
+        if (data.me === matches.target)
+          return { alarmText: output.tankLaserOnYou() };
+        return { infoText: output.avoidTankLaser() };
+      },
+    },
+    {
+      id: 'Zadnor Diablo Aetheric Explosion',
+      netRegex: NetRegexes.startsUsing({ source: 'The Diablo Armament', id: '5CC6', capture: false }),
+      response: Responses.aoe(),
+    },
+    {
+      id: 'Zadnor Diablo Ultimate Psuedoterror',
+      // This is triggered on Diabolic Gate with a delay, so it gives an extra +4 seconds.
+      netRegex: NetRegexes.startsUsing({ source: 'The Diablo Armament', id: '5C9F', capture: false }),
+      delaySeconds: 35,
```

Comment:
Is there some information about the cast that can make this number less magical?

Reply:
Unfortunately, no.  This is a series of a bunch of attacks (with the same ability id) and afterwards you may have to book it across the entire arena to get under.  This is a "timeline trigger without a timeline".